src/bluesky_stomp/messaging.py

```diff
@@ -224,7 +236,12 @@ def _send_bytes(
             )
         if correlation_id:
             headers = {**headers, CORRELATION_ID_HEADER: correlation_id}
-        self._conn.send(headers=headers, body=message, destination=destination)  # type: ignore
+
+        tracer = get_tracer("_send_bytes")
+
+        with tracer.start_as_current_span("_send_bytes"):
+            propagate_context_in_stomp_headers(headers)
+            self._conn.send(headers=headers, body=message, destination=destination)  # type: ignore
```

Comment:
it would be good to use add_span_attributes to capture the contenet of the headers at this point as well so that we can see these as well as the message content